### PR TITLE
fix: set preferCurrentTab to false bye default

### PR DESCRIPTION
**Pull request by cloudwebrtc**
close https://github.com/livekit/client-sdk-flutter/issues/337

### CODE REVIEW

File: lib/src/track/options.dart [hunk 1]
```diff
@@ -102,15 +102,16 @@ class ScreenShareCaptureOptions extends VideoCaptureOptions {
   final bool captureScreenAudio;
 
   /// for browser only, if true, will capture current tab.
-  final bool preferCurrentTab;
+  @Deprecated('Deprecated parameter no longer works in chrome m115+')
```
Comment (davidzhao) [inline lib/src/track/options.dart:105]: I don't think this is deprecated.. what do you mean it no longer works?

Comment (davidzhao) [reply lib/src/track/options.dart:105]: I think the problem is that we've defaulted it to true.. the default should be false.

Comment (cloudwebrtc) [reply lib/src/track/options.dart:105]: ah, okay.

**Review by davidzhao — APPROVED** [review]
lg

**Pull request merged**